sql/expression/function/aggregation/window_framer.go

```diff
@@ -316,32 +319,24 @@ func (f *rowFramerBase) Interval() (sql.WindowInterval, error) {
 // frame:  {0,3},   {1,3}, {2,3}, {3,5},   {3,5},   {4,5}
 // rows:   [0,1,2], [1,2], [2],   [4,4,5], [4,4,5], [5]
 type rangeFramerBase struct {
-	idx                          int
-	partitionStart, partitionEnd int
-	frameStart, frameEnd         int
-	partitionSet                 bool
-
-	// reference expression for boundary calculation
-	orderBy sql.Expression
-
-	// boundary arithmetic on [orderBy] for range start value
-	// is set unless [unboundedPreceding] is true
-	startInclusion sql.Expression
-	// boundary arithmetic on [orderBy] for range end value
-	// is set unless [unboundedFollowing] is true
-	endInclusion sql.Expression
```

Comment:
Why not keep these comments around? Seems like they could be helpful for new readers of this code.